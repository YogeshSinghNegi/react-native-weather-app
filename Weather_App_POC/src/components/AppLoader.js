import * as AppColors from '../utils/AppColors';

import { ActivityIndicator, StyleSheet, View } from  'react-native';

import React from 'react';

const AppLoader = (props) => {
    return (
        <View style = {Styles.activity}>
            <ActivityIndicator style = {{color: AppColors.darkColor}}/>
        </View>
    );
};

const Styles = StyleSheet.create({
    activity: {
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center'
    }
})

export default AppLoader;