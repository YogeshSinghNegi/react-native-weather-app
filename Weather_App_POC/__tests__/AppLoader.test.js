import 'react-native';

import AppLoader from '../src/components/AppLoader';
import React from 'react';
import renderer from 'react-test-renderer';

it('<AppLoader/> renders correctly', () => {
  const tree = renderer.create(
    <AppLoader />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});