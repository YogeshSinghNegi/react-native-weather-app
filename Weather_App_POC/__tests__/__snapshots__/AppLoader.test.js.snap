// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppLoader/> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="small"
    style={
      Object {
        "color": "#303F9F",
      }
    }
  />
</View>
`;
